x-pack/plugins/security_solution/public/detections/components/rules/rule_preview/loading_histogram.tsx

```diff
@@ -29,14 +29,7 @@ export const LoadingHistogram = () => {
         <EuiFlexItem grow={1}>
           <LoadingChart size="l" data-test-subj="preview-histogram-loading" />
         </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <>
-            <EuiSpacer />
-            <EuiText size="s" color="subdued">
-              <p>{i18n.QUERY_PREVIEW_DISCLAIMER}</p>
-            </EuiText>
-          </>
-        </EuiFlexItem>
+        <EuiSpacer />
```

Comment:
I realize this is not part of this PR but is the "loading" text (passed as `subtitle` prop to HeaderSection) when creating the Histogram intended? Looks a little weird to me, especially having the loading icon already there:

![image](https://user-images.githubusercontent.com/5354282/189416976-a21da7de-e6cc-4c3b-8314-9e4ca7e4efcc.png)

Reply:
Yes it is intended and was like this since a long time (at least I see the same in 7.17.5 version). Though, we can ask what design team thinks about it and whether we need to update this behaviour. @yiyangliu9286 What do you think about this one?